modules/heroku/variables.tf

```
@@ -70,7 +70,7 @@ variable "worker_dyno_quantity" {
 variable "postgresql_plan" {
   type        = string
   nullable    = true
-  default     = "basic"
+  default     = "essential-1"
```

```suggestion
  default     = "essential-0"
```

My concern about the old `mini` was that it had a limit of 10,000 rows. `essential-0` doesn't appear to have any such limit, so let's just go with the cheapest plan as a default.